server/controllers/pg.js

```diff
@@ -96,7 +96,8 @@ const postCategory = (calendarId, category, callback) => {
     const client = await pool.connect()
     try {
       const result = await client.query(query.postCategory, [calendarId, category]);
-      callback(null, 'Category Posted');
+      let category_id = result.rows[0].category_id;
+      callback(null, category_id);
```

Comment:
ok, so as I'm understanding it -- the pg POST query will return the category_id of the posted new category.